module/VuFindSearch/src/VuFindSearch/Backend/ProQuestFSG/Connector.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * Class for accessing ProQuestFSG search API
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Andrew Nagy 2008.
```

Comment:
Might want to update this copyright statement. You can certainly give Andrew credit in the author list if this is heavily derived from his original code, but I suspect this is largely your work and justifies a 2025 date.

Reply:
Yeah that was copy/paste mistake from the SRU connector.  I'm leaving this all at 2024 tho.